src/test/java/com/atlassian/httpclient/apache/httpcomponents/ApacheAsyncHttpClientTest.java

```diff
@@ -164,55 +163,50 @@ public class ProxyTestHandler extends AbstractHandler {
         final String realm = "test_realm";
 
         @Override
-        public void handle(
-                String target,
-                org.eclipse.jetty.server.Request jettyRequest,
-                HttpServletRequest request,
-                HttpServletResponse response)
-                throws IOException, ServletException {
+        public boolean handle(
```

Comment:
changes looks good. 
But I wonder if it would be better to simply use Servlet API filter to have some implementation of this not depending on Jetty core API (which may change soon again for 12.1.x)